src/eme.js

```diff
@@ -348,6 +348,7 @@ const standardizeKeySystemOptions = (keySystem, keySystemOptions) => {
   }
 
   const isFairplay = isFairplayKeySystem(keySystem);
+  const isPlayready = keySystem === 'com.microsoft.playready' || keySystem === 'com.microsoft.playready.recommendation';
```

Comment:
Should this include the `recommendation.3000` variant as well?

Reply:
Not in this state. It should be something like:
```
const isPlayready = [
    'com.microsoft.playready',
    'com.microsoft.playready.recommendation',
    'com.microsoft.playready.recommendation.3000'
    ].includes(keySystem)'
```
or a little bit less safe `keySyste.startsWith('com.microsoft.playready')` for .3000 to work. I wanted to add that up but I never got to that.